src/Components/Directory.Build.targets

```diff
@@ -1,3 +1,11 @@
 ﻿<Project>
+  <PropertyGroup Condition="'$(UsingMicrosoftNETSdkBlazorWebAssembly)' == 'true' and '$(BuildNodeJS)' != 'false'">
```

Comment:
BuildNodeJs is `` in some cases, so this check needs to explicitly check against `false`